Epsilon.Host.Frontend/.eslintrc

```diff
@@ -1,31 +1,31 @@
 {
```

Comment:
Did you only do formatting on this file, it's difficult to see what changed

Reply:
prettier

Reply:
changed